pages/writing/scaling_support.mdx

```diff
@@ -0,0 +1,808 @@
+          <MediaBody>
+            <Box
+              as="div"
+              color="colorText"
+              backgroundColor="colorBackgroundUser"
+              borderRadius="borderRadius20"
+              borderBottomRightRadius="borderRadius0"
+              fontSize="fontSize30"
+              lineHeight="lineHeight20"
+              padding="space30"
+            >
+              Hey 👩🏻‍🎤! Welcome to design system help!
+              <br />
+              📖 Start by{" "}
+              <Link href="//paste.twilio.design/" passHref>
+                <Anchor>exploring our docs</Anchor>
+              </Link>
+              <br />❓ Still need help? <Link href="//github.com/twilio-labs/paste/discussions">
+                <Anchor>Start a discussion</Anchor>
+              </Link>
+              <br />
+              🕐 Expect a 1 day response time
+              <br />
+              🚨 Extremely urgent request, post in Slack!
+            </Box>
+            <Text
+              as="div"
+              color="colorText"
+              fontSize="fontSize50"
+              marginTop="space10"
+              textAlign="right"
+            >
+              🤖
+            </Text>
+          </MediaBody>
+        </MediaObject>
+      </Box>
+    </Stack>
+  );
+};
+const triggerEmergency = (props) => {
+  const [displayProp, setDisplayProp] = React.useState("none");
+  return (
+    <>
+      <Box display="flex" justifyContent="flex-end">
+        <MediaObject as="div">
+          <MediaBody>
+            <Box
+              as="div"
+              color="colorText"
+              backgroundColor="colorBackgroundUser"
+              borderRadius="borderRadius20"
+              borderBottomRightRadius="borderRadius0"
+              fontSize="fontSize30"
+              lineHeight="lineHeight20"
+              padding="space30"
+              maxWidth="size30"
+            >
+              Hi! Post your question on{" "}
+              <Link href="//github.com/twilio-labs/paste/discussions" passHref>
+                <Anchor>GitHub</Anchor>
+              </Link>
+              ! If this is an extremely urgent request press the emergency button,
+              and a DSYS team member will get back to you!
+              <Box marginTop="space20">
+                <Button
+                  variant="primary"
+                  size="small"
+                  onClick={() => {
+                    setDisplayProp("flex");
+                  }}
+                >
+                  Emergency
+                </Button>
+              </Box>
+              <Box display={displayProp} marginTop="space40">
+                Aye aye! Paging a team member now. Hang tight!
+              </Box>
+            </Box>
+            <Text
+              as="div"
+              color="colorText"
+              fontSize="fontSize50"
+              marginTop="space10"
+              textAlign="right"
+            >
+              🤖
+            </Text>
```

Comment:
Nit: spacing between the robot and chat bubble below it could be a little spacier.